tools/clang/test/DXC/shader_model_update.test

```diff
@@ -0,0 +1,9 @@
+
+
+// RUN: %dxc  %S/Inputs/smoke.hlsl  /D "semantic = SV_Position" /T vs_6_0 /Zi /Qembed_debug /DDX12 /Fo %t.define.cso
+
+// dxc.exe shader model upgrade
```

Comment:
I recognize this is how this test was described before, but it just seems to verify that we can disassemble rather than anything to do with SM upgrades

Reply:
Good catch.
The real test is removed in an old change in hcttestcmds.com.
Will fix it.